spark/dl/src/main/scala/com/intel/analytics/bigdl/parameters/AllReduceParameter.scala

```diff
@@ -184,63 +219,75 @@ class AllReduceParameter[T: ClassTag](id: Long, partitionNum: Int,
             BlockManagerWrapper.unlock(blockId)
             pid
           } catch {
-            case t : Throwable =>
+            case t: Throwable =>
               logger.error("Error: " + ExceptionUtils.getStackTrace(t))
               throw t
           }
         }
       }
-    })
+    }
     syncPool.invokeAll(sgThreads.asJava)
 
     val length = taskSize + (if (partitionId < extraSize) 1 else 0)
     val poolSize = Engine.default.getPoolSize
     val innerTaskSize = length / poolSize
     val innerExtraSize = length % poolSize
     val availableTask = if (innerTaskSize == 0) innerExtraSize else poolSize
-    Engine.default.invokeAndWait2((0 until availableTask).map(tid => () => {
-      val innerStart = tid * innerTaskSize + math.min(innerExtraSize, tid)
-      val innerLength = innerTaskSize + (if (tid < innerExtraSize) 1 else 0)
-      params.reduce((l, r) => l.add(r.bytes(innerStart, innerLength), innerStart,
-        innerLength))
-      tid
-    }))
-
+    Engine.default.invokeAndWait2 {
+      (0 until availableTask).map { tid =>
+        () => {
+          val innerStart = tid * innerTaskSize + math.min(innerExtraSize, tid)
+          val innerLength = innerTaskSize + (if (tid < innerExtraSize) 1 else 0)
+          params.reduce { (l, r) =>
+            l.add(r.bytes(innerStart, innerLength), innerStart, innerLength)
+          }
+          tid
+        }
+      }
+    }
     params.head.deCompress(gradientPartition)
   }
 
+  /**
+   * Slice gradients learned from this partition of data into chunks, and mark each chunk to be sent
+   * to the appropriate parameter node, and put it in the block manager.
+   *
+   * @param parameter A Tensor that contains gradients computed on the entire model on a single
+   *                  partition of data.
+   */
   def putGradients(parameter: Tensor[T]): Unit = {
     var pid = 0
-    val bm = SparkEnv.get.blockManager
-    require(parameterBuffer != null)
+    require(parameterBuffer != null, )
```

Comment:
It's a typo. It's better to have a message in all `require` instances. I'll add one, thanks!